scripts/linters/js_ts_linter.py

```diff
@@ -1166,7 +1201,8 @@ def _lint_js_and_ts_files(self):
         if num_files_with_errors:
             for error in result_list:
                 python_utils.PRINT(error)
-                summary_messages.append(error)
+            eslint_messages = self._get_trimmed_error_messages(result_list[0])
```

Comment:
Why [0]? Needs justification and perhaps better naming to explain what result_list[0] actually is.